.gitignore

```diff
@@ -135,3 +135,6 @@ dmypy.json
 /metadata/**/*
 metadata
 /ssh
+
+# DS Store
+echo .DS_Store >> ~/.gitignore_global
```

Comment:
Can you remove this change from the PR? This should be local and not saved to the repo. Thanks.